promgen/templates/promgen/vue/silence_modal.html

```diff
@@ -0,0 +1,108 @@
+<div id="silenceModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="silenceModalLabel">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" @click="hideModal('silenceModal')" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```

Comment:
```suggestion
        <button type="button" class="close" @click="state.show = false" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```

Reply:
👍 Thank you!!  All updated in https://github.com/line/promgen/pull/523/commits/22fd06ccc136494f33bf547967913693ffcc0946

Reply:
> 👍 Thank you!! All updated in [22fd06c](https://github.com/line/promgen/commit/22fd06ccc136494f33bf547967913693ffcc0946)

All changes reviewed and look fine.

Also tested it in my local machine and works fine.

Looks good to me. Could you please rebase+squash? Then we can approve it and merge it.

Good job!!!